week-8-practise/1-code-refactoring.js

```diff
@@ -8,28 +8,31 @@
  * There may be both syntax and logical errors present.
  */
 
-var reversed;
-let capitilised;
 
 // Function that reverses and capitlises text
-exports function reverseAndCaptilise(text) 
-  var capitlised = text.toUpper()
-  reversed = capitilised.split('').reverse().join('') 
 
-  return reversed
-  }
+export function reversesAndCaptilises(text) {
+  return text
+    .split("")
+    .reverse()
+    .join("")
+    .toUpperCase();
+}
 
 // Function that returns text with every other letter capitlised
+
 export function everyOtherLetterCapitilised(text) {
-  for (i==0; i++; i<text.length) {
-      if (i/2==0) {
-       text[i] = text[i].toUpperCase() 
-    }    
-  } 
+  text = text.split("");
+  for (i = 0; i < text.length; i++) {
+    if (i % 2 === 0) {
+      text[i] = text[i].toUpperCase();
+    }
+  }
+  return text.join("");
 }
 
-export  function lowerCase(text) {
-  let text = text.toLowerCase()
+//Faucntion that returns text with every other letter in small case
 
-  return text
-}
+export function everyOtherLetterTolowerCase(text) {
```

Comment:
This is incorrect - the function is supposed to make all the letters in a word lower case (I appreciate this may not have been clear without comments)